src/service.js

```diff
@@ -30,19 +23,16 @@ const GameService = new Lang.Class({
             'com.endlessm.CodingGameService.Service',
             '/com/endlessm/CodingGameService/Service',
             null);
-    },
+    }
 
-    startMission: function(mission) {
-        this._service.call_start_mission(mission, null, Lang.bind(this, function(source, result) {
+    startMission(mission) {
+        this._service.call_start_mission(mission, null, (source, result) => {
```

Comment:
Surprising - I thought that arrow functions did not bind `this`?

Reply:
Binding `this` is the whole shtick of arrow functions :smile: